### ISSUE TITLE
Publish fails for package dependent on `fs-events`

### ISSUE THREAD

**Issue by tmeasday** [meteor/meteor#7634]
I was testing https://github.com/meteor/meteor/pull/7608, and followed:

```bash
# create a package
meteor create --release 1.4.1-rc.0 --package tmeasday:binary-test && cd binary-test;

# edit package.js to add `Npm.depends({ fsevents: '1.0.14' });

# test publication w/ old meteor works but complains about publish-with-arch
meteor publish --create

# bump version number

# now try and publish with new release
meteor publish --release 1.4.1-rc.0
```

I get

```
Error: Cannot find module '../'
at Function.Module._resolveFilename (module.js:325:15)
at Function.Module._load (module.js:276:25)
at Module.require (module.js:353:17)
at require (internal/module.js:12:17)
at Object.<anonymous> (/private/tmp/test-binary/.npm/package/node_modules/.temp-1aqk8tg/node_modules/fsevents/node_modules/.bin/node-pre-gyp:15:20)
at Module._compile (module.js:409:26)
at Object.Module._extensions..js (module.js:416:10)
at Module.load (module.js:343:32)
at Function.Module._load (module.js:300:12)
at Function.Module.runMain (module.js:441:10)

npm ERR! Darwin 15.6.0
npm ERR! argv "/Users/tom/.meteor/packages/meteor-tool/.1.4.1-rc.0.3aqn5a++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/bin/node"
"/Users/tom/.meteor/packages/meteor-tool/.1.4.1-rc.0.3aqn5a++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/bin/npm" "rebuild" "--no-bin-links" "--update-binary"
npm ERR! node v4.4.7
npm ERR! npm  v3.10.5
npm ERR! code ELIFECYCLE
npm ERR! fsevents@1.0.14 install: `node-pre-gyp install --fallback-to-build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the fsevents@1.0.14 install script 'node-pre-gyp install --fallback-to-build'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the fsevents package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-pre-gyp install --fallback-to-build
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs fsevents
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls fsevents
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /private/tmp/test-binary/.npm/package/node_modules/.temp-1aqk8tg/npm-debug.log
module.js:327
throw err;
```

**Comment by benjamn** [comment]
I'm not able to reproduces this. Does it help if you remove the `/private/tmp/test-binary/.npm/` before publishing again?

**Comment by tmeasday** [comment]
Yeah, that works. I guess we close this? Maybe we should add documentation about doing this somewhere?

**Comment by benjamn** [comment]
Had this happen to me with `1.4.1-rc.3` and came up with a solution: when we copy an npm package directory to rebuild it, we won't copy `node_modules/.bin/{node-gyp,node-pre-gyp}`, so `npm rebuild` won't get confused by `node_modules/.bin/node-pre-gyp`.

**Comment by benjamn** [comment]
This is fixed in Meteor 1.4.1.
